src/components/ContentPreview/ContentPreview.js

```diff
@@ -132,6 +132,10 @@ const RETRY_COUNT = 3; // number of times to retry network request for a file
 const MS_IN_S = 1000; // ms in a sec
 const PREVIEW_LOAD_METRIC_EVENT = 'load';
 
+const LoadableSidebar = AsyncLoad({
+    loader: () => import(/* webpackMode: "lazy", webpackChunkName: "content-sidebar" */ '../ContentSidebar'),
```

Comment:
Where does this chunk name get set? I don't see it in our webpack config.

Reply:
This sets the output chunk name https://webpack.js.org/api/module-methods/#magic-comments

Reply:
For both LoadableSidebar and LoadableContentOpenWith, should we have them in their own files in their respective parent folders exported?